### PR TITLE
Student response

### PR DESCRIPTION
**Pull request by SULAIMAN-100**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Provide a short description of the change you are proposing and the reasons for it.

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/components/MentorViewSubmission.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        });
+    }
+  };
+
+  useEffect(() => {
+    axios
+      .get(`/api/get-submissions/${student_id}`)
+      .then((response) => {
+        setCardData(response.data);
+        console.log(response.data.map((id) => id.id));
```
Comment (Hedyeh65) [inline client/src/components/MentorViewSubmission.js:60]: please remove the console.log

File: client/src/components/ModuleDropDown.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
 import LessonsDropDown from "./LessonsDropDown";
-const ModuleDropDown = () => {
+const ModuleDropDown = ({ id }) => {
   const [subject, setSubject] = useState();
   const [select, setSelect] = useState();
-
+  console.log(id);
```
Comment (Hedyeh65) [inline client/src/components/ModuleDropDown.js:7]: remove this console.log please

File: client/src/components/SubmissionCard.js [hunk 3]
```diff
@@ -1,22 +1,99 @@
...
+                mentorName.find((m) => m.id === card.mentor_id).surname +
+                " On " +
+                card.title
+              ) : (
+                <p className="waiting-feedback">Waiting for feedback ...</p>
+              )}
+            </Card.Title>
+            <div className="card-color">
+              <div id="card-date">
+                {" "}
```
Comment (Hedyeh65) [inline client/src/components/SubmissionCard.js:55]: remove this bracket please

File: server/db.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ if (process.env.DATABASE_URL) {
   config = {
     host: "localhost",
     database: "compass",
-    password: "************",
+    password: "12133405",
```
Comment (Hedyeh65) [inline server/db.js:21]: remove your password please

**Review by Hedyeh65 — CHANGES_REQUESTED** [review]

**Review by FatimaAbdimalik — CHANGES_REQUESTED** [review]

**Review by FatimaAbdimalik — APPROVED** [review]

**Pull request merged**